app/javascript/src/components/Dashboard/Tasks/DeleteModal.jsx

```diff
@@ -1,37 +1,21 @@
 import React from "react";
-import { Modal } from "neetoui";
+import { Alert } from "neetoui";
 
 export default function DeleteAlert({ onClose, handleDelete }) {
   return (
-    <Modal
+    <Alert
       isOpen
-      size="small"
-      autoHeight
-      showFooter
+      title="Delete Task"
+      message="Are you sure you want to delete the task?"
+      icon="text-red-500 ri-alarm-warning-fill ri-lg"
```

Comment:
Do we need this anymore? If not, please remove it.

Reply:
We do not. was trying to make it the same as the Figma design.
Removed title message and icon